course-definition.yml

```
@@ -310,6 +310,22 @@ stages:
 
       The tester will assert that the stdout of your program matches the format above.
 
+      Reference character names:
```

```suggestion
      Here's a list of all the single-character tokens along with their types:
```

(more friendly, + "names" might not be accurate, types is consistent with the book)